react/src/components/Input.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import styled from 'styled-components';
 
 interface InputProps {
-    value: number;
+    value: any;
```

Comment:
I think this can be number or string. We might just default to string and let handler functions take care of conversions if necessary.